jb/itch5/generate_inside.hpp

```diff
@@ -25,12 +25,11 @@ namespace itch5 {
  * (before the any output is generated).
  * @returns true if the inside is affected by the change, false otherwise.
  */
-template <typename duration_t>
+template <typename duration_t, typename book_type>
```

Comment:
Please document the new template parameter with a @tparam book_type as described above.